### DISCUSSION TITLE
2024-06-12 双周会

### DISCUSSION THREAD

**Discussion by jeasonstudio** [ant-design/ant-design-web3#947] [中文社区双周会]
上次周会：https://github.com/ant-design/ant-design-web3/discussions/917

**Comment by yutingzhao1991** [comment]
- 讨论下风格切换的设计需求
<img width="900" alt="image" src="https://github.com/ant-design/ant-design-web3/assets/1061968/1a49e884-2ba9-40f2-892a-e27f9bb822cf">

这部分可以结合首页的那部分一起做一个新的设计，体现我们主题切换的能力

**Reply by jeasonstudio** [reply]
connectkit: https://docs.family.co/connectkit/try-it-out
![screenshot-VInMkbin@2x](https://github.com/ant-design/ant-design-web3/assets/17971291/57ebe234-010f-4ea5-9e21-9ec75efa4882)

thirdweb: https://thirdweb.com/dashboard/connect/playground
![screenshot-jWTb2XI1@2x](https://github.com/ant-design/ant-design-web3/assets/17971291/279b56f8-013f-4a27-b996-3f0d89e76b36)

**Comment by jeasonstudio** [comment]
PR:
* https://github.com/ant-design/ant-design-web3/pull/937
* https://github.com/ant-design/ant-design-web3/pull/933 (补一下 test coverage）
* https://github.com/ant-design/ant-design-web3/pull/916
* https://github.com/ant-design/ant-design-web3/pull/876
